packages/core/src/Renderer.ts

```diff
@@ -615,6 +499,246 @@ export class Renderer extends AbstractRenderer
         return this.plugins.extract;
     }
 
+    /** Collection of plugins */
+    get plugins(): IRendererPlugins
+    {
+        return this._plugin.plugins;
+    }
+
+    /** The number of msaa samples of the canvas. */
+    get multisample(): MSAA_QUALITY
+    {
+        return this._multisample.multisample;
+    }
+
+    /**
+     * Same as view.width, actual number of pixels in the canvas by horizontal.
+     *
+     * @member {number}
+     * @readonly
+     * @default 800
+     */
+    get width(): number
+    {
+        return this._view.view.width;
+    }
+
+    /**
+     * Same as view.height, actual number of pixels in the canvas by vertical.
+     * @default 600
+     */
+    get height(): number
+    {
+        return this._view.view.height;
```

Comment:
I agree that this is kinda weird `view.view`. I think ViewSystem makes a lot of sense actually and probably would like to keep that. What about renaming the `view` property in ViewSystem instead? Maybe something like `element` (or alternatively `canvas`). 

```
renderer._view.element.width
renderer._view.element.height
```

"View" is much more general term and includes things like resolution, the screen-size, resizing etc. Where the actual rendering surface (HTMLCanvasElement, NodeCanvas, etc) can be more specific.

Reply:
like that! renamed to element.